### PR TITLE
Add new item

### PR DESCRIPTION
**Pull request by nosensus**
1. Added api request for method POST.
2. Implemented hook - useTodoListItemCreate.

### CODE REVIEW

File: src/api/TodoListApi.ts [hunk 1]
```diff
@@ -1,7 +1,19 @@
...
   return await fetch(`${API_URL}/items`);
 }
 
-export { getTodoList }
+const postTodoListItem = async (newItem: TodoItem) => {
+  await fetch(`${API_URL}`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+      'Accept': 'application/json',
```
Comment (adizka) [inline src/api/TodoListApi.ts:13]: Not sure why you need these headers, json is by default for all requests.

File: src/hooks/useTodoListItemCreate.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      console.log(error);
+    } finally {
+      queryTodoList();
+    }
+  };
+
+  return { todoListItemCreate };
+};
+
+export { useTodoListItemCreate };
```
Comment (adizka) [inline src/hooks/useTodoListItemCreate.tsx:26]: You created a hook with only one method inside, you could do it inside the component itself.

**Review by adizka — APPROVED** [review]

File: src/hooks/useTodoListItemCreate.tsx (hunk 2 above)
Comment (nosensus) [reply src/hooks/useTodoListItemCreate.tsx:26]: Component includes Add and Edit methods inside. If I move inside than Edit in the feature component will bloat.

File: src/api/TodoListApi.ts (hunk 1 above)
Comment (nosensus) [reply src/api/TodoListApi.ts:13]: I did it because I thought use real Database. That's why included headers at once.

**Pull request merged**